meetings/2023-04-06.md

```diff
@@ -0,0 +1,96 @@
+* JM: eP made the case that a decent solution is capability URLs. cURLs does not require discussion over poken generation or authn. Creating an agreement between NS and NR. Probably more accurately the subscriber. In the future we might have authn scheme that generates tokens. Right now, the plan for implementation work to have capability URLs. That's where I am right now. eP can correct.
+* eP: Seems accurate. We can defer for unspecified for solving the problem using another authn scheme comparable to Solid OIDC. It is added convenience. Implementation doesn't need to be blocked until we resolve that topic.
+* eP: We use cURLs use. It is valid for streaming HTTP as well. There are other valid reasons ot have it to make explicit that it is used. For HTTP at least, receive cURL, so don't have tho do authn. It is already there - useful to have it explicit so that NR knows how to connect to the notification channel. There are some previous efforts to have authn directly on the endpoint over websocket protocol. it'd be beneficial if websocket channel doesn't have cURL but it can also use different authn scheme that's explicitly set for the channel.
+* eP: In CSS, provides a package for some of products of classes in the Notifications Protocol. If Acme has is using CSS, they'd have an IdP built in part of CSS. They take advantage from all those pieces being together. That's not the case for other implementations so it is not the reliable way of doing it.
+* eP: In #504, NS is always acting as server-side client. CSS has the convenience for implicit --- it generates token without asking the user to authn/z. If implementations were to be separate, then all problems in 504 described would come to place.
+* SC: {Summary of the shortcoming of Solid-OIDC for this picture}
+* eP: I'd like to understand if the token is used by NR .. what stops it from using it elsewhere?
+* JM: Assuming-ly, the token receiver is same entity that subscribed to it. Is there case where the receiver wouldn't be the same entity?
+* JM: Can still constrain the token with multi-embedded credentials. Sign the token by the server, saying this is intended audience.. e.g., NS could generate a token put in the audience of the NR and send it over. No DPOP required. Then NR can't re-use it at all.
+* eP: Constraining the audience.
+* SC: Are we considering alternatives to move towards?
+* JM: As an implementer, just do cURL for now.. the trade-off is with cURL hs the same security-boundaries. The only ... NR to have something like UUID in the URL. Instead of having one nice URL - however they're receiving it. In the future we might want to also have new authn scheme so that NR can have URL setup instead of cURL.
+* JM: Is cURL described in the current spec?
+* eP: metnions authn issue
+* eP: It is not presented to the user - used at runtime but not exposed. In Some systems, webhook for example, it can be copied. here it is in the subscription response but not to human interaction.
```

Comment:
```suggestion
* eP: mentions authn issue
* eP: It is not presented to the user — used at runtime but not exposed. In some systems, webhook for example, it can be copied. Here it is in the subscription response but not to human interaction.
```